docker/README.md

```diff
@@ -10,4 +10,4 @@ You can specify the injected documentation repository as mounted volume under th
 
 ## Start the container via the compose file
 
-You can start the container manual oder use can also use the `docker-compose up -d` command to run it inside the daemon mode and open the documentation inside your browser under `localhost:4567`. 
\ No newline at end of file
+You can start the container manual oder use can also use the `docker-compose up -d` command to run it inside the daemon mode and open the documentation inside your browser under `localhost:4567`. 
```

Comment:
What does the word "oder" refer to? Is this a typo?
Looks like this line had 2 sentences smashed together at some point, and it was never fixed up...?

Reply:
Cool! I should have figured that out :-)
@ZPascal , would you mind translating this to English and then we can close off this PR? Thanks!